src/org/rascalmpl/parser/gtd/SGTDBF.java

```diff
@@ -63,6 +64,7 @@
  */
 public abstract class SGTDBF<P, T, S> implements IGTD<P, T, S> {
 	private final static int DEFAULT_TODOLIST_CAPACITY = 16;
+	private static final String PARAM_ENABLE_CACHE = "enable-cache";
```

Comment:
In this PR parse graph flattening is configured using a query parameter. This will be removed in the final version but currently very useful for debugging.